### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by shanel262**
Updated readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
...
+> A [Seneca.js](https://github.com/senecajs/) a seneca-auth plugin
+
+# seneca-twitter-auth
+
+[![npm version][npm-badge]][npm-url]
+[![Build Status][travis-badge]][travis-url]
+[![Dependency Status][david-badge]][david-url]
+[![Gitter chat][gitter-badge]][gitter-url]
+
+Lead Maintainer: [Mircea Alexandru](https://github.com/mirceaalexandru)
```
Comment (mihaidma) [inline README.md]: don't think Mircea is lead maintainer here, please remove

File: README.md [hunk 2]
```diff
@@ -63,5 +84,18 @@ The JSON object provided for this action contains following data from Twitter lo
...
 
+## Contributing
+The [Senecajs org](https://github.com/senecajs/) encourage open participation. If you feel you can help in any way, be it with
+documentation, examples, extra testing, or new features please get in touch.
+
 ## License
-Copyright Mircea Alexandru and other contributors 2016, Licensed under [MIT](./LICENSE).
\ No newline at end of file
+Copyright Mircea Alexandru and other contributors 2016, Licensed under [MIT](./LICENSE).
+
```
Comment (mihaidma) [inline README.md]: please check original author, it could be Nicolas

File: README.md [hunk 3]
```diff
@@ -63,5 +82,18 @@ The JSON object provided for this action contains following data from Twitter lo
...
+## Contributing
+The [Senecajs org](https://github.com/senecajs/) encourage open participation. If you feel you can help in any way, be it with
+documentation, examples, extra testing, or new features please get in touch.
+
 ## License
-Copyright Mircea Alexandru and other contributors 2016, Licensed under [MIT](./LICENSE).
\ No newline at end of file
+Copyright Nicolas Herment and other contributors 2016, Licensed under [MIT](./LICENSE).
+
+[npm-badge]: https://badge.fury.io/js/seneca-local-auth.svg
```
Comment (ckiss) [inline README.md]: seneca-local-auth?!

File: README.md [hunk 4]
```diff
@@ -63,5 +80,14 @@ The JSON object provided for this action contains following data from Twitter lo
...
+## Contributing
+The [Senecajs org](https://github.com/senecajs/) encourage open participation. If you feel you can help in any way, be it with
+documentation, examples, extra testing, or new features please get in touch.
+
 ## License
-Copyright Mircea Alexandru and other contributors 2016, Licensed under [MIT](./LICENSE).
\ No newline at end of file
+Copyright Nicolas Herment and other contributors 2016, Licensed under [MIT](./LICENSE).
+
+[david-badge]: https://david-dm.org/mirceaalexandru/seneca-twitter-auth.svg
```
Comment (ckiss) [inline README.md]: senecajs instead of mirceaalexandru

**Comment by ckiss** [comment]
LGTM

**Pull request merged**